run_unittests.py

```diff
@@ -2613,20 +2627,36 @@ def test_cross_file_system_paths(self):
             raise unittest.SkipTest('system crossfile paths not defined for Windows (yet)')
 
         testdir = os.path.join(self.common_test_dir, '1 trivial')
-        cross_content = textwrap.dedent("""\
-            [binaries]
-            c = '/usr/bin/cc'
-            ar = '/usr/bin/ar'
-            strip = '/usr/bin/ar'
+        if is_sunos():
+            cross_content = textwrap.dedent("""\
+                [binaries]
+                c = '/usr/bin/gcc'
+                ar = '/usr/bin/gar'
+                strip = '/usr/bin/gar'
 
-            [properties]
+                [properties]
 
-            [host_machine]
-            system = 'linux'
-            cpu_family = 'x86'
-            cpu = 'i686'
-            endian = 'little'
-            """)
+                [host_machine]
+                system = 'sunos'
+                cpu_family = 'x86'
+                cpu = 'i686'
+                endian = 'little'
+                """)
+        else:
+            cross_content = textwrap.dedent("""\
+                [binaries]
+                c = '/usr/bin/cc'
+                ar = '/usr/bin/ar'
+                strip = '/usr/bin/ar'
+
+                [properties]
+
+                [host_machine]
+                system = 'linux'
+                cpu_family = 'x86'
+                cpu = 'i686'
+                endian = 'little'
+                """)
```

Comment:
I'd rather use `str.format()` to pass the minor differences between these two strings than copy the whole thing.

Reply:
Done.